plugins/inputs/opcua/README.md

```diff
@@ -16,12 +16,22 @@ Plugin minimum tested version: 1.16
   ## OPC UA Endpoint URL
   # endpoint = "opc.tcp://localhost:4840"
   #
+  ## The service set to use to retrieve/receive the values from the OPC UA server
+  # connection_service = "request"
+  # connection_service = "subscribe"
+  #
   ## Maximum time allowed to establish a connect to the endpoint.
   # connect_timeout = "10s"
   #
   ## Maximum time allowed for a request over the estabilished connection.
   # request_timeout = "5s"
   #
+  ## The interval at which the server should at least update its monitored items
+  ## https://reference.opcfoundation.org/v104/Core/docs/Part4/5.13.1/
```

Comment:
Is this URL something that should be present in every configuration file? This could maybe be better explained in the readme file.